static/javascripts/components/survey_card.jsx

```diff
@@ -288,6 +316,18 @@ var CheckboxQuestion = React.createClass({
     addOption: function(){
         var options = this.state.options;
         var numOptions = this.state.numOptions;
+
+        if (numOptions>0){
+            for(var i=0; i<options.length;i++){
+              if (options[i].title==""){
+              alert("Complete all the current options before adding one");
```

Comment:
I don't know if I think this is necessary. When I make surveys I like to click the button till I have enough options and then fill it out. It just seems inconvenient without great reason.